hack/tools.go

```diff
@@ -1,10 +0,0 @@
-//go:build tools
-// +build tools
-
-// This package imports things required by build scripts, to force `go mod` to see them as dependencies.
-package tools
-
-import (
-	_ "github.com/onsi/ginkgo/ginkgo"
```

Comment:
I see the integration tests are failing, and I think it is because you removed this. Can you just remove the line with code-generator import?